<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Use Student List</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="cancle_number_spinBox"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="studentlist_lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="username_lineEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="download_pushButton">
     <property name="text">
      <string>Download Studentlist</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="password_lineEdit">
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Cancel Number</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Password</string>
     </property>
     <property name="buddy">
      <cstring>password_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="last_download_label">
     <property name="text">
      <string>last download: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Moodle Username</string>
     </property>
     <property name="buddy">
      <cstring>username_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="open_pushButton">
     <property name="text">
      <string>Open...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>LDAP</string>
     </property>
     <property name="buddy">
      <cstring>download_pushButton</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Student List Path</string>
     </property>
     <property name="buddy">
      <cstring>studentlist_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="config_label">
     <property name="text">
      <string>Config file at config.ini</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Config</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>username_lineEdit</tabstop>
  <tabstop>password_lineEdit</tabstop>
  <tabstop>download_pushButton</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>studentlist_lineEdit</tabstop>
  <tabstop>open_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
